tasks/README.md

```diff
@@ -5,5 +5,86 @@ This directory contains the tasks that are part of this benchmark.
 
 Name | Summary | Category
 ---- | ----------- | --------
-`task1001_squad1.1_question_generation` | Generating guestions (based on SQuAD 1.1) | Question Generation  
-`task1002_squad1.1_answer_generation` | Generating answers to SQuAD 1.1 questions | Answer Generation
```

Comment:
add this task as well, they were part of the initial repo

Reply:
Do I have to add it in the REAEDME.md?
or 
Do I have to add the files as well?

Reply:
both

Reply:
@krishna-8 did you add those 2 files? I can see that there are conflicts because you didn't add the files, only their names in the README

Reply:
I have added the tasks, there is still a conflict with the readme file.

Reply:
thanks, will take care of that